src/extension/importer/rest/src/main/java/org/geoserver/importer/rest/ImportBaseController.java

```diff
@@ -98,7 +104,21 @@ ImportTransform transform(ImportTask task, Integer transformId, boolean optional
             try {
                 tx = (ImportTransform) task.getTransform().getTransforms().get(transformId);
             } catch (NumberFormatException e) {
+                LOGGER.log(
```

Comment:
Since the exception message is exactly the same, could you use a multicatch and avoid the code duplication?
See https://docs.oracle.com/javase/8/docs/technotes/guides/language/catch-multiple.html